website/cue/reference/components/sources/base/docker_logs.cue

```diff
@@ -45,7 +45,7 @@ base: components: sources: docker_logs: configuration: {
 			[global_host_key]: https://vector.dev/docs/reference/configuration/global-options/#log_schema.host_key
 			"""
 		required: false
-		type: string: default: "host"
+		type: string: {}
```

Comment:
Is there no default value anymore?

Reply:
It'll default to `log_schema.host_key` as mentioned in the option description. The field itself just doesn't have an explicit default.

Reply:
(or rather it defaults to "null" which will cause it to fallback to the value of `log_schema.host_key`)